frontend/packages/console-shared/src/components/dashboard/status-card/HealthItem.tsx

```diff
@@ -48,7 +48,11 @@ const HealthItem: React.FC<HealthItemProps> = React.memo(
             )}
           </span>
           {state !== HealthState.LOADING && detailMessage && (
-            <SecondaryStatus status={detailMessage} className="co-status-card__health-item-text" />
+            <SecondaryStatus
+              status={detailMessage}
+              className="co-status-card__health-item-text"
+              dataStatusID={`${title.toLowerCase().replace(/\s/g, '-')}-secondary-status`}
```

Comment:
```suggestion
              dataStatusID={`${title}-secondary-status`}
```

Reply:
@bipuladh This was suggested by @vbnrh a while back in a sync up https://github.com/openshift/console/pull/9823#discussion_r699355252

Reply:
Should I drop this change?

Reply:
I would rather that the title is in sync with the test-id. So if the title is kept in a constant then when it's changed in one place there will not be any need to change it in the test.

Reply:
Initial idea was to use the title to make the secondary status selectable. I personally prefer having attributes like this in lower case format without spacing. Makes it clean and easier to read.